### PR TITLE
[11.0][MIG] account_invoice_force_number

### PR DESCRIPTION
**Pull request by OpenCode**
Migrate account_invoice_force_number to 11.0

### CODE REVIEW

**Review by melroy89 — APPROVED** [review]

File: account_invoice_force_number/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
```
Comment (melroy89) [inline account_invoice_force_number/__manifest__.py]: You can remove the utf-8 lines from the top in Python3.

**Review by melroy89 — CHANGES_REQUESTED** [review]
Remove utf-8 lines at the top.

File: account_invoice_force_number/tests/test_account_force_number.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
```
Comment (melroy89) [inline account_invoice_force_number/tests/test_account_force_number.py]: You can remove the utf-8 lines from the top in Python3.

**Comment by melroy89** [comment]
Just be sure to safe your files as utf-8 encoding.

**Comment by OpenCode** [comment]
@danger89 done

**Review by melroy89 — APPROVED** [review]

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM

**Review by tafaRU — APPROVED** [review]

**Pull request merged**